### PR TITLE
[RAPTOR-4506] skip predict env var for fit.sh 

### PR DESCRIPTION
**Pull request by jlpolit**
# This repository is public. Do not put here any private DataRobot or customer's data: code, datasets, model artifacts, .etc.

### CODE REVIEW

**Comment by jlpolit** [comment]
jarvis check this

File: public_dropin_environments/python3_sklearn/fit.sh [hunk 1]
```diff
@@ -17,6 +17,10 @@ if [ -f "${weights}" ]; then
     CMD="${CMD} --row-weights-csv ${weights}"
 fi
 
+if [ -n "${SKIP_PREDICT}" ]; then
+  CMD = "${CMD} --skip-predict"
```
Comment (jmbannon) [inline public_dropin_environments/python3_sklearn/fit.sh:21]: nit: doesn't follow batch syntax (which has no spaces)

**Review by jmbannon — APPROVED** [review]

**Comment by yakov-g** [comment]
DRUM supports taking params directly from env var, take a look here:
https://github.com/datarobot/datarobot-user-models/blob/2ca9da2828ec810a32a3a5681d7243f341dca86d/custom_model_runner/datarobot_drum/drum/main.py#L118

If you properly add support for `SKIP_PREDICT` env var, you won't need to change fit.sh scripts.

**Comment by jlpolit** [comment]
> DRUM supports taking params directly from env var, take a look here:
> https://github.com/datarobot/datarobot-user-models/blob/2ca9da2828ec810a32a3a5681d7243f341dca86d/custom_model_runner/datarobot_drum/drum/main.py#L118
> 
> If you properly add support for `SKIP_PREDICT` env var, you won't need to change fit.sh scripts.

Great- will look into doing that! thanks

**Comment by yakov-g** [comment]
Could you also clean up changelog please:
- put date: `#### [1.4.12] - 2021-01-19
- put date: `#### [1.4.11] - 2021-01-14
- create an new record for [1.4.13
 

Thanks

**Comment by jlpolit** [comment]
closing in favor of https://github.com/datarobot/datarobot-user-models/pull/281

**Comment by jlpolit** [comment]
> Could you also clean up changelog please:
> 
> * put date: `#### [1.4.12] - 2021-01-19
> * put date: `#### [1.4.11] - 2021-01-14
> * create an new record for [1.4.13
> 
> Thanks

@yakov-g  will do - fyi I opened a new PR with the new approach, will apply those changes there

**Pull request closed without merging**